crates/sargon/src/radix_connect/wallet_interaction/dapp_wallet_interaction/wallet_to_dapp/success_response/pre_authorization/subintent_response.rs

```diff
@@ -116,33 +123,8 @@ mod tests {
 
     #[test]
     fn json_success() {
-        // Test roundtrip for sample (whose timestamp doesn't include milliseconds).
         assert_json_roundtrip(&SUT::sample());
-
-        // Test encoding of sample_other (whose timestamp includes 123 milliseconds).
-        let result =
-            serde_json::to_string_pretty(&SUT::sample_other()).unwrap();
-        let json = r#"
```

Comment:
I think this test was better, we should assert against an actual representation of the json that would be sent. Doing the json roundtrip can be successful, while the json that is sent to the dApp does not match the RDT expectation.

Reply:
makes sense, bringing it back